.gitignore

```diff
@@ -11,7 +11,6 @@ pids
 lib-cov
 
 # dist
-dist/
```

Comment:
Could you return dist folder back to .gitignore? We do not commit bundled artifacts to source control

Reply:
yeap, on it, I removed it and committed dist only because I wanted to use the fork in my projects to manually make sure my changes work as expected